build-scripts/substitute-local-appservices.gradle

```diff
@@ -41,7 +41,7 @@ configurations.all { config ->
                         def name = dependency.requested.module
                         // Although there are a number of app-services group names which might be configured,
                         // our local publish flow always uses exactly `org.mozilla.appservices`
-                        dependency.useTarget([group: 'org.mozilla.appservices', name: name, version: version])
+                        dependency.useTarget([group: group, name: name, version: version])
```

Comment:
At the very least the comment should change here - but I made this change to *fix* autopublish with nightlies - a-c was looking for a package named `org.mozilla.appservices.something` and we always autopublish with this name - so our auto-published version was never found.